package.json

```diff
@@ -597,7 +597,7 @@
         "kudu-api": "^1.4.1",
         "opn": "^5.1.0",
         "request": "^2.83.0",
-        "vscode-azureappservice": "~0.8.0",
```

Comment:
vscode-azureappservice's package has this:


    "engines": {

        "vscode": "^1.20.0"

    }

You may need to update engine here, too.  I know we'll need to when I update it to the new UI package.